components/Link.js

```diff
@@ -1,14 +1,31 @@
 import { html } from 'https://unpkg.com/rplus-production@1.0.0';
 import shouldPreventDefault from '../utils/shouldPreventDefault.js';
 
-export default ({ href, onClick, innerText }) => {
+const pushState = url => history.pushState(null, null, url);
+
+/** Link component props:
+ * href = anchor href
+ * innerText = text to display between anchor tags
```

Comment:
This is traditionally called `children` as technically you could pass other components in here, not just text.

Reply:
Happy to rename `innerText` to `children` if you think that works better!

I'm neutral, as I think you're right, but also a `children` prop wouldn't be an obvious place for me to just put text. I wonder if there's a way to easily convey both?

Reply:
This is convention really; as supposed to personal preference. Components usually have the signature `(props, children)` (reinforced by JSX) where children can be text or vdom node.